CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # Change Log
 
+## [0.28.0](https://github.com/eclipse/lemminx/milestone/44?closed=1) (May 27, 2024)
```

Comment:
This milestone has not yet been created, and has no closed issues in it. This link is dead

Reply:
Yup. All the new issues were placed under the previous milestone, which was already released. I don't have commit rights here so can't move them into a new 0.28.0 one but it should be just a matter of taking the issues/PRs in this chanagelog and moving them there.

Reply:
Okay, forgot you don't have commit rights here, I can do that, one second

Reply:
okay, I think I got it